examples/oft-solana/README.md

```diff
@@ -198,31 +198,37 @@ sh -c "$(curl -sSfL https://release.solana.com/v1.17.31/install)"
 
 ### Create the Solana OFT
 
-For OFT:
+:information_source: In all the OFT patterns listed below, the SPL token's Mint Authority is set to the **Mint Authority Multisig**, which always has the **OFT Store** as a signer. The multisig is fixed to needing 1 of N signatures.
+
+:information_source: For **OFT** and **OFT Mint-And-Burn Adapter**, you have the option to specify additional signers through the `--additional-minters` flag. If you choose not to, you must pass in `--only-oft-store true`, which means only the **OFT Store** will be a signer for the \_Mint Authority Multisig\*.
+
+:warning: If you choose to go with `--only-oft-store`, you will not be able to add in other signers/minters or update the Mint Authority.
```

Comment:
Probably good time to add a warning about Freeze Authority as well. If you choose `--only-oft-store` you can't renounce Freeze Authority.